### DISCUSSION TITLE
Check for: Reserved Words

### DISCUSSION THREAD

**Discussion by aptMattKoe** [StefanMaron/BusinessCentral.LinterCop#26] [New Rule]
A function should not be called Validate, SetRange or an Field should not be named "Count".

**Comment by fvet** [comment]
Launched similar idea at https://github.com/microsoft/AL/issues/5141

**Comment by Arthurvdv** [comment]
If anyone has a list of reserved words for functions and/or fields, I’d be more than happy to create a rule for it.
